pkg/datapath/loader/template.go

```diff
@@ -167,10 +167,10 @@ func elfMapSubstitutions(ep datapath.Endpoint) map[string]string {
 		}
 	}
 
-	// The policy map is only used for the host endpoint if per-endpoint
-	// routes and the host firewall are enabled.
-	if !ep.IsHost() ||
-		(option.Config.EnableEndpointRoutes && option.Config.EnableHostFirewall) {
+	// Populate the policy map if the host firewall is enabled regardless of the per-endpoint route setting
```

Comment:
Nit: This should be wrap like other lines. Not sure why checkpatch doesn't complain. cc @qmonnet

Reply:
Because it's Go code and we're not passing it through checkpatch.pl?

Reply:
Ah, of course! :facepalm: